ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -152,8 +152,14 @@ else
 	then
 		if param compare SYS_AUTOCONFIG 1
 		then
-			# Wipe out params except RC*, flight modes, total flight time, accel cal, gyro cal, next flight UUID
-			param reset_nostart RC* COM_FLTMODE* LND_FLIGHT_T_* TC_* CAL_ACC* CAL_GYRO* COM_FLIGHT_UUID
+			if param compare CBRK_KEEP_MAGCAL 382716
+			then
+				# Wipe out params except RC*, flight modes, total flight time, next flight UUID, accel cal, gyro cal, AND mag cal
+				param reset_nostart RC* COM_FLTMODE* LND_FLIGHT_T_* TC_* COM_FLIGHT_UUID CAL_ACC* CAL_GYRO* CAL_MAG*
```

Comment:
I'm wondering if CAL_MAG**x**_ROT should still be reset.

Reply:
Hum, this circuit breaker is to avoid any sensor config when you know that nothing but software airframe will change. 

For this reason, I think it is best not to reset external mag rotation when the circuit breaker is set.
On the contrary, in the normal case, external mags are probably mounted differently on the new airframe, so it is safer to reset CAL_MAGx_ROT.

Does that make sense?